.github/workflows/build-and-test.yml

```diff
@@ -22,47 +22,29 @@ jobs:
     with:
       runsonlabels: '["macOS", "self-hosted", "spezi"]'
       scheme: SpeziBluetooth-Package
+      swiftVersion: 6
```

Comment:
Remove in case we are moving to Swift 6 by default anyways. I am not 100% sure this even does a difference here?